lib/rules/max-empty-lines/index.js

```diff
@@ -23,47 +21,28 @@ const rule = function (max) {
     if (!validOptions) {
       return
     }
-
+    
+    // `//`-comments are need to be processed by postcss-scss
+    // To do that we have to use .toString()
     const rootString = root.toString()
```

Comment:
`// single line comment` and `/* single line comment */` - should report on difference index, but your code seems report on equals, btw `root.source.input.css` is good practice, another syntax can have other comments directive.